gix-date/src/time/format.rs

```diff
@@ -1,37 +1,22 @@
-use time::macros::format_description;
-
 use crate::{
     time::{CustomFormat, Format},
     Time,
 };
 
 /// E.g. `2018-12-24`
-pub const SHORT: CustomFormat = CustomFormat(format_description!("[year]-[month]-[day]"));
+pub const SHORT: CustomFormat = CustomFormat("%Y-%m-%d");
 
 /// E.g. `Thu, 18 Aug 2022 12:45:06 +0800`
-pub const RFC2822: CustomFormat = CustomFormat(format_description!(
-    "[weekday repr:short], [day] [month repr:short] [year] [hour]:[minute]:[second] [offset_hour sign:mandatory][offset_minute]"
-));
+pub const RFC2822: CustomFormat = CustomFormat("%a, %d %b %Y %H:%M:%S %z");
 
 /// E.g. `Thu, 8 Aug 2022 12:45:06 +0800`. This is output by `git log --pretty=%aD`.
-pub const GIT_RFC2822: CustomFormat = CustomFormat(format_description!(
-    "[weekday repr:short], \
-     [day padding:none] \
-     [month repr:short] \
-     [year] \
-     [hour]:[minute]:[second] \
-     [offset_hour sign:mandatory][offset_minute]"
-));
+pub const GIT_RFC2822: CustomFormat = CustomFormat("%a, %-d %b %Y %H:%M:%S %z");
```

Comment:
This crate finally comes full circle :): initially `time` would have a format string like this, then it moved to `format_description!` with more verbose declarations and optimized efficiency. Now we go back to simple format strings which are probably just the right choice here.

Reply:
Hah! Although Jiff's `strtime` APIs are quite fast. At least in Jiff's benchmarks, they're faster than `time`'s `format_description!` parsers.

But yeah I don't really like the `strtime` APIs myself, but they are handy and it seems like almost everyone is familiar with them.